tequilapi/endpoints/list_identities_test.go

```diff
@@ -0,0 +1,38 @@
+package endpoints
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListIdentities(t *testing.T) {
+
+	req := httptest.NewRequest("GET", "/irrelevant", nil)
+	resp := httptest.NewRecorder()
+
+	mockIdm := newManager("0x000000000000000000000000000000000000000A")
+	handlerFunc := ListIdentitiesEndpointFactory(mockIdm).ListIdentities
+	handlerFunc(resp, req, httprouter.Params{})
```

Comment:
I thing 3rd parameter could be optional. Should try

Reply:
it's not optional in this router library